css/components.css

```diff
@@ -0,0 +1,269 @@
+  --info: #2196f3;
+  --success: #4caf50;
+  --warning: #ff9800;
+  --error: #f44336;
+  --secondary-hover: #637cbb;
+  --info-hover: #6daee4;
+  --success-hover: #76b878;
+  --warning-hover: #ebab4c;
+  --error-hover: #fa756c;
+}
+
+body{
+  margin: 0;
+  font-family: 'Roboto', sans-serif;
+}
+
+.sidebar{
+  min-height: 100vh;
+  border-right: 1px solid black;
+  position: fixed;
+  left: 0;
+  top: 0;
+}
+
+.sidebar > ul  {
+  padding: 1rem;
+}
+
+.sidebar > ul li {
+  margin-bottom: 1rem;
+  list-style: none;
+}
+
+.sidebar > ul li a{
+  display: block;
+}
+
+.main-content{
+  /* min-height: 100vh; */
+  margin-left: 10rem;
+}
+
+.main-content > ul li {
+  border-bottom: 1px solid  gray;
+}
+
+.code-div{
+  margin: 1rem 0;
+}
+
+.bg-primary{
+  background-color: var(--primary);
+}
+
+.bg-secondary{
+  background-color: var(--secondary);
+}
+
+.bg-info{
+  background-color: var(--info);
+}
+
+.bg-success{
+  background-color: var(--success);
+}
+
+.bg-warning{
+  background-color: var(--warning);
+}
+
+.bg-error{
+  background-color: var(--error);
+}
+
+.bg-secondary:hover{
+  background-color: var(--secondary-hover);
+}
+
+.bg-info:hover{
+  background-color: var(--info-hover);
+}
+
+.bg-success:hover{
+  background-color: var(--success-hover);
+}
+
+.bg-warning:hover{
+  background-color: var(--warning-hover);
+}
+
+.bg-error:hover{
+  background-color: var(--error-hover);
+}
+
+.avatar-small{
+    width: 1rem;
+    height: 1rem;
+    border-radius: 1rem;
+}
+
+.avatar-medium{
+    width: 2rem;
+    height: 2rem;
+    border-radius: 2rem;
+}
+
+.avatar-large{
+    width: 4rem;
+    height: 4rem;
+    border-radius: 4rem;
+}
+
+.alert {
+    border-radius: 5px;
+    width: max-content;
+    height: max-content;
+    text-align: left;
+    padding: 0.5rem;
+    overflow: hidden;
+    margin: 1rem 0;
+  }
+
+  .alert-text {
+    display: inline-block;
+    margin: 0.5rem 0.5rem 0.5rem 0;
+    font-weight: bold;
+    color: white;
+  }
+
+  .alert-btn {
+    color: black;
+    width: 1.5rem;
+    height: 1.5rem;
+    border-radius: 1.5rem;
+    border: 0;
+    outline: 0;
+    font-weight: bold;
+  }
+
+  .alert-info {
+    background-color: var(--info);
+  }
+  
+  .alert-success {
+    background-color: var(--success);
+  }
+  
+  .alert-error {
+    background-color: var(--error);
+  }
+  
+  .alert-warning {
+    background-color: var(--warning);
+  }
+  .alert-btn-info {
+    background-color: var(--info);
+  }
+  
+  .alert-btn-info:hover {
+    background-color: var(--info-hover);
+  }
+  
+  .alert-btn-success {
+    background-color: var(--success);
+  }
+  
+  .alert-btn-success:hover {
+    background-color: var(--success-hover);
+  }
+  
+  .alert-btn-error {
+    background-color: var(--error);
+  }
+  
+  .alert-btn-error:hover {
+    background-color: var(--error-hover);
+  }
+  
+  .alert-btn-warning {
+    background-color: var(--warning);
+  }
+  
+  .alert-btn-warning:hover {
+    background-color: #ebab4c;
+  }
+
+
+  .badge{
```

Comment:
Just a suggestion, you can add separators here for css classes related to each sections.
Example: 

--badge--


--avatar--


--alert--

This can be helpful for you to debug the code later. I have seen a lot of people following this as well.